pages/Useful Utilities/App-Clients.md

```diff
@@ -11,6 +11,37 @@ on the Discord ToS.
 - [gtkcord4](https://github.com/diamondburned/gtkcord4) is a Discord client written in GTK4.
 While it does infringe on Discord's ToS, it's relatively safe and doesn't rely on any webview technologies.
 
+# Spotify
+
+It has been reported that Spotify does not follow window rules. This is possibly
+because the client sets is class _after_ the window has opened, thus making it "immune"
+to windowrules. An alternative to Spotify's GUI client is
+[spotify-tui](https://github.com/Rigellute/spotify-tui) which can be launched in a terminal
+with a custom class. While limited in functionality, it is quite powerful and could be
+preferred over the GUI client if windowrules are a dealbreaker for you.
```

Comment:
```suggestion
Spotify does not follow window rules. This is because the client sets is class _after_
the window has opened, thus making it "immune" to windowrules. An alternative to
Spotify's GUI client is [spotify-tui](https://github.com/Rigellute/spotify-tui) which can be
launched in a terminal with a custom class. While limited in functionality, it is quite
powerful and could be preferred over the GUI client.
```